python/ray/train/torch/train_loop_utils.py

```diff
@@ -39,11 +39,17 @@
 
 
 @PublicAPI(stability="stable")
-def get_device() -> Union[torch.device, List[torch.device]]:
+def get_device() -> torch.device:
     """Gets the correct torch device configured for this process.
 
-    Returns a list of devices if more than 1 GPU per worker
-    is requested.
+    Returns the torch device for the current worker. If more than
+    1 GPU is requested per worker, returns the first device.
```

Comment:
"with the minimum device index" (is that correct?)

Reply:
yes. let me update it